linkcheck/cookies.py

```diff
@@ -22,12 +22,12 @@
     from http.cookiejar import split_header_words
 except ImportError: # Python 2
     from cookielib import split_header_words
-try: # Python 3
-    from http.client import HTTPMessage
-except ImportError: # Python 2
-    from httplib import HTTPMessage
```

Comment:
what was wrong with this approach? why are we switching to the email module? this at least warrants a comment in the source.

Reply:
I would like to know too!

Reply:
Commit comment added in #293.